include/lexy/dsl/error.hpp

```diff
@@ -86,7 +86,7 @@ LEXY_CONSTEVAL auto prevent(Pattern pattern)
 namespace lexyd
 {
 template <typename Tag, typename Pattern>
-struct _try : rule_base
+struct rule_try : rule_base
```

Comment:
Can you use something like `_try_` or `_try_r`?

The type names should be short as error messages are already long enough, and I like the leading underscore as it hides them from auto completion.